src/app/communities/link.ts

```diff
@@ -1,4 +1,5 @@
 export interface Link {
+  id: string;
```

Comment:
It is just a precaution but I am afraid there has to be `number` type instead of `string` as there is `Long` type for the id field of `Link` on the server side https://github.com/T-Systems-MMS/skoop-server/blob/master/src/main/java/com/tsmms/skoop/community/Link.java#L20.